sql/metadata.go

```diff
@@ -0,0 +1,146 @@
+package sql
+
+import (
+	"database/sql"
+	"fmt"
+	"sqlflow.org/gomaxcompute"
+	"strconv"
+	"strings"
+)
+
+type Metadata struct {
+	odpsConfig *gomaxcompute.Config
+	table      string
+	featureMap *featureMap
+	columnInfo *map[string]*columnSpec
+}
+
+func (meta *Metadata) GetColumnInfo(featureColumns map[string][]featureColumn) (map[string]*columnSpec, error) {
+	allKeys := make([]string, 0)
+	for _, fcs := range featureColumns {
+		fields := GetAllKeys(fcs)
+		allKeys = append(allKeys, fields...)
+	}
+	columns, err := meta.GetSparseColumnInfo()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
We use [logrus](github.com/sirupsen/logrus) to log errors
https://github.com/joyyoj/sqlflow/blob/8149bd4b679d75033b533a1d9c272823334fec57/sql/executor.go#L149